tests/tests/da_compression.rs

```diff
@@ -68,7 +110,19 @@ async fn can_fetch_da_compressed_block_from_graphql() {
 
     let block = client.da_compressed_block(block_height).await.unwrap();
     let block = block.expect("Unable to get compressed block");
-    let _: VersionedCompressedBlock = postcard::from_bytes(&block).unwrap();
+    let block: VersionedCompressedBlock = postcard::from_bytes(&block).unwrap();
+
+    let db = &srv.shared.database;
+    let mut tx_inner = db.off_chain().clone().into_transaction();
+    let db_tx = DecompressDbTx {
+        db_tx: DbTx {
+            db_tx: &mut tx_inner,
+        },
+        onchain_db: db.on_chain().view_at(&0u32.into()).unwrap(),
+    };
+    let decompressed = decompress(compression_config, db_tx, block).await.unwrap();
+
+    assert!(decompressed.transactions.len() == 2);
```

Comment:
Might as well check that the second tx matches the one submitted.

Reply:
Some fields will differ, as the returned tx would have to be executed to generate full values